apps/web/src/pages/get-started/consts/DelayUseCase.const.tsx

```diff
@@ -25,7 +27,7 @@ export const DelayUseCaseConst: OnboardingUseCase = {
         return (
           <StepDescription>
             <StepText>Novu pre-built workflow with a digest node.</StepText>
-            <Link children={' Customize '} href={'https://mantine.dev/core/timeline/'} />
+            <Link href={ROUTES.INTEGRATIONS}> Integration store</Link>
```

Comment:
The value here should be actually `Customize`, [ref](https://www.figma.com/file/hlsUxjq2kxUU7aVBf2QyGu/Onboarding--experience?node-id=3608%3A25001&mode=dev)
But the link (button) would call the API endpoint to create the workflow from the blueprint constant trigger id.

Reply:
Shoot thank you, good catch